type-definitions/ts-tests/tslint.json

```diff
@@ -1,10 +0,0 @@
-{
-  "extends": "dtslint/dtslint.json",
```

Comment:
Despite having `dtslint` in its name, this file simply holds a list of `tslint` rules, reference: https://github.com/microsoft/dtslint/blob/master/dtslint.json